dynadjust/make_dynadjust_gcc_ubuntu.sh

```diff
@@ -91,7 +91,7 @@ fi
 
 cmake ../
 
-make -j
+make -j2
```

Comment:
I had to adjust the number of parallel processes, as I was getting memory errors due to Make spinning off more processes than I had memory. I've set it to two, since Travis build environment only has [two cores](https://docs.travis-ci.com/user/reference/overview/#virtualisation-environment-vs-operating-system).

Reply:
Looks good @jmettes . I think this is an important QA tool and 15 minutes is not too long.